js/app.js

```diff
@@ -0,0 +1,170 @@
+//Step 1 Declare Variables
+//Set Questions
+var questions = [{
+    //Question 1
+    question: "If you were strolling along this street, where would you be?",
```

Comment:
One or the other for everything, unless there is a very clear pattern. In this case, the "choices" switches back and forth, which makes it look inconsistent.